src/BuildScriptGenerator/Helpers/PreAndPostBuildCommandHelper.cs

```diff
@@ -4,42 +4,36 @@
 // --------------------------------------------------------------------------------------------
 
 using System.IO;
+using Microsoft.Oryx.Common;
 
 namespace Microsoft.Oryx.BuildScriptGenerator
 {
     internal static class PreAndPostBuildCommandHelper
     {
-        public static (string preBuildCommand, string postBuildCommand) GetPreAndPostBuildCommands(
-            ISourceRepo sourceRepo,
-            EnvironmentSettings settings)
```

Comment:
nit: this seem to be a breaking change. do we still be supporting old settings or  from now on users will mention/keep build.env in their repo instead if having app_settings? how does this fit into azure function and the vs online and may be GitHub actions?

Reply:
Take a look at my other comments. They will give an idea of the changes here.